release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -870,6 +870,18 @@ For more information about these configuration fields, see xref:../networking/cl
 The Telemetry and the Insights Operator collects telemetry on IPsec connections. For more information, see xref:../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#showing-data-collected-from-the-cluster_showing-data-collected-by-remote-health-monitoring[Showing data collected by Telemetry].
 
 [id="ocp-4-16-registry_{context}"]
+[id="overlapping-ip-configuration-multi-tenant-networks-whereabouts"]
+==== Overlapping IP configuration for multi-tenant networks with Whereabouts
+
+Previously, it was not possible to configure the same CIDR range twice and to have the Whereabouts CNI plugin assign IP addresses from these ranges independently. This limitation caused issues in multi-tenant environments where different groups might need to select overlapping CIDR ranges. 
```

Comment:
Does `CIDR` need to be spelled out in its first mention, or is it well known enough to stand on its own?

Reply:
I've never seen it spelled it before. I think if you're poking around the networking docs it's "common knowledge"